service/src/main/java/uk/gov/hmcts/reform/fpl/json/deserializer/ChildSelectorDeserializer.java

```diff
@@ -0,0 +1,52 @@
+package uk.gov.hmcts.reform.fpl.json.deserializer;
+
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.core.TreeNode;
+import com.fasterxml.jackson.databind.DeserializationContext;
+import com.fasterxml.jackson.databind.JsonDeserializer;
+import com.fasterxml.jackson.databind.node.NullNode;
+import com.fasterxml.jackson.databind.node.TextNode;
+import org.springframework.boot.jackson.JsonComponent;
+import uk.gov.hmcts.reform.fpl.model.order.generated.selector.ChildSelector;
+
+import java.io.IOException;
+
+@JsonComponent
+public class ChildSelectorDeserializer extends JsonDeserializer<ChildSelector> {
+    @Override
+    public ChildSelector deserialize(JsonParser parser,
```

Comment:
hmm, I'll be honest I was just thinking ChildSelector.stream() in the controller or a service 🤔 

Not sure if this is overkill

Reply:
Removed